assets/js/localization.js

```diff
@@ -354,6 +354,7 @@ function localizeStrings(stringsFresh /*: boolean */) {
                         }
                         else {
                             elem.html(text);
+                            elem.attr('dir', rtlLanguages.indexOf(locale) === -1 ? 'ltr' : 'rtl');
```

Comment:
You think https://github.com/JPJPJPOPOP/apertium-html-tools/blob/3ac77a33ff60c0623eeeb29b22932345812bf1ee/assets/js/localization.js#L391 could be pulled out and used? I think there are a couple other places that could be easily replaced as well.

Reply:
(the code does seem to work though!)